samples/functions/default_arguments_in_method.jakt

```diff
@@ -0,0 +1,13 @@
+/// Expect:
+/// - output: "42\n"
+
+enum S {
+    X
+    function my_func(this, anon def: i64 = 42) {
```

Comment:
I guess that the change is specifically fixing enum default arguments?
As there is `samples/functions/default_arguments/method.jakt` that seems to work currently.
If so probably best to mention enum in the commit message/ sample name.

Reply:
True, just noticed that as well when writing on discord

Reply:
Updated the commit message now, thanks